cypress/tests/core/basic/linkIntegrityView.js

```diff
@@ -0,0 +1,457 @@
+  beforeEach(() => {
+    cy.autologin();
+    cy.visit('/');
+  });
+
+  it('As an Editor I can see if my document is linked somewhere via teaser block', () => {
+    cy.createContent({
+      contentType: 'Document',
+      contentTitle: 'Document that is linked',
+      contentId: 'document-linked',
+    });
+
+    cy.createContent({
+      contentType: 'Document',
+      contentTitle: 'Document that is linking via teaser block',
+      contentId: 'document-linking',
+      bodyModifier(body) {
+        body.blocks['abc'] = {
+          '@type': 'teaser',
+          description: '',
+          href: [
+            {
+              '@id': '/document-linked',
+              '@type': 'Document',
+              Description: '',
+              Title: 'Document linked',
+              effective: '2023-08-14T22:00:42+00:00',
+              getObjSize: '0 KB',
+              hasPreviewImage: null,
+              head_title: null,
+              image_field: '',
+              mime_type: 'text/plain',
+              title: 'Document linked',
+            },
+          ],
+          title: 'Document linked',
+        };
+        body.blocks_layout.items.push('abc');
+
+        return body;
+      },
+    });
+
+    cy.visit('/document-linked/links-to-item');
+    cy.get('tbody > :nth-child(2) > :nth-child(1)').contains(
+      'Document that is linking via teaser block',
+    );
+  });
+
+  it('As an Editor I can see if my image is linked somewhere via override image in teaser block', () => {
+    cy.createContent({
+      contentType: 'Document',
+      contentTitle: 'Document that is linked',
+      contentId: 'document-linked',
+    });
+
+    cy.createContent({
+      contentType: 'Image',
+      contentTitle: 'Image that is linked',
+      contentId: 'image-linked',
+    });
+
+    // TODO: get the content creation with image override to work here
+    cy.createContent({
+      contentType: 'Document',
+      contentTitle: 'Document that is linking via teaser block',
+      contentId: 'document-linking',
+      bodyModifier(body) {
+        body.blocks['abc'] = {
+          '@type': 'teaser',
+          description: '',
+          href: [
+            {
+              '@id': '/document-linked',
+              '@type': 'Document',
+              Description: '',
+              Title: 'Document linked',
+              effective: '2023-08-14T22:00:42+00:00',
+              getObjSize: '0 KB',
+              hasPreviewImage: null,
+              head_title: null,
+              image_field: 'image',
+              mime_type: 'text/plain',
+              title: 'Document linked',
+              // preview_image: [
+              //   {
+              //     '@id': '/image-linked',
+              //     '@type': 'Image',
+              //     image_field: 'image',
+              //     Title: 'Image linked',
+              //     Type: 'Image',
+              //   },
+              // ],
```

Comment:
@jackahl @sneridagh Is the comment here the reason for the continued test failure?

Reply:
@jackahl Could you please take a look, so we can close this? The strange thing is that https://github.com/kitconcept/fzj-internet/pull/1488 is passing, and this one, do not.

Reply:
@davisagli it should not be. This is commented out, as the `createContent()` function seems to have trouble to add preview images. There for I am adding the image "by hand" afterwards.

Reply:
But I can confirm the image is added as it can be seen in cypress after this line has been run 
![Screenshot 2023-11-27 at 15 53 26](https://github.com/plone/volto/assets/44289551/42f59c6e-830c-4773-ba1f-1f4dcc5f045c)
https://github.com/plone/volto/pull/5111/files/5c54795589b8166d9aa91cb602fe7eba96b90041#diff-3fc8e2fbc20dc1ed7d495f717dc8c3cc8b774e3b8050a11f5655297eb3885db4R321